src/lunamary/modelPerson/Device.java

```diff
@@ -2,13 +2,15 @@
 
 public class Device {
 
-    private String type; // App, Movil, Email
-    private String identifier; // number or email
+    private String type;
+    private String identifier;
```

Comment:
Looks like you need identifiers on most of your entities. Probably is a good idea to create a base abstract class called BaseModel or something that has a unique identifier you can use java.util.UUID to achieve that